manifests/federation/0.0.7/federation.v0.0.7.clusterserviceversion.yaml

```diff
@@ -12,6 +12,367 @@ metadata:
+          "kind": "Cluster",
+          "metadata": {
+            "name": "cluster-name"
+          },
+          "spec": {
+            "authInfo": {},
+            "kubernetesApiEndpoints": {
+              "serverEndpoints": [
+                {
+                  "clientCIDR": "0.0.0.0/0",
+                  "serverAddress": "https://cluster-name.example.com:6443"
+                }
+              ]
+            }
+          }
+        },
+        {
+          "apiVersion": "core.federation.k8s.io/v1alpha1",
+          "kind": "FederatedCluster",
+          "metadata": {
+            "name": "cluster-name"
+          },
+          "spec": {
+            "clusterRef": {
+              "name": "cluster-name"
+            },
+            "secretRef": {
+              "name": "<name-of-secret>"
+            }
+          }
+        },
+        {
+          "apiVersion": "types.federation.k8s.io/v1alpha1",
+          "kind": "FederatedDeployment",
+          "metadata": {
+            "name": "hello"
+          },
+          "spec": {
+            "placement": {
+              "clusterNames": [
+                "cluster2",
+                "cluster1"
+              ]
+            },
+            "template": {
+              "apiVersion": "extensions/v1beta1",
+              "kind": "Deployment",
+              "metadata": {
+                "name": "hello"
+              },
+              "spec": {
+                "replicas": 1,
+                "template": {
+                  "metadata": {
+                    "labels": {
+                      "app": "hello"
+                    }
+                  },
+                  "spec": {
+                    "containers": [
+                      {
+                        "name": "hello",
+                        "image": "openshift/hello-openshift:latest",
+                        "ports": [
+                          {
+                            "containerPort": 80
+                          }
+                        ]
+                      }
+                    ]
+                  }
+                }
+              }
+            }
+          }
+        },
+        {
+          "apiVersion": "types.federation.k8s.io/v1alpha1",
+          "kind": "FederatedClusterRole",
+          "metadata": {
+            "name": "test-clusterrole"
+          },
+          "spec": {
+            "template": {
+              "rules": [
+                {
+                  "apiGroups": [
+                    "*"
+                  ],
+                  "resources": [
+                    "*"
+                  ],
+                  "verbs": [
+                    "*"
+                  ]
+                }
+              ]
+            },
+            "placement": {
+              "clusterNames": [
+                "cluster2",
+                "cluster1"
+              ]
+            }
+          }
+        },
+        {
+          "apiVersion": "types.federation.k8s.io/v1alpha1",
```

Comment:
We don't want to have this in examples because it's something you generate - not a type the operator actually gives you directly.

Reply:
Removing everything else leaves just the Cluster and FederatedCluster examples, is there anything else I should add?